compiler/rustc_target/src/spec/targets/aarch64_pc_windows_gnullvm.rs

```diff
@@ -17,7 +17,7 @@ pub(crate) fn target() -> Target {
         metadata: TargetMetadata {
             description: Some("ARM64 MinGW (Windows 10+), LLVM ABI".into()),
             tier: Some(2),
-            host_tools: Some(false),
+            host_tools: Some(true),
```

Comment:
Question: wait, cc @mati865, I think this was intentional for a reason I can't immediately recall, right?

Reply:
Gnullvm didn't ship host tools initially and then this change was missed in https://github.com/rust-lang/rust/pull/140772
Thanks for fixing that.